src/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{30CAB353-089E-4294-B23B-F2DD1D945654}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
There are actually a few other projects where we need this constant since there are other projects where we include SerializationTypes.cs file. Do you mind doing a quick search and adding the line to the rest? If I counted right, there were about 10 projects that included this file